vendor/github.com/aws/aws-sdk-go/aws/awsutil/prettify.go

```diff
@@ -1,113 +0,0 @@
-package awsutil
```

Comment:
I just figured since it wasn't used anywhere with the lines removed here that the lib should be removed. I don't have anything against it.